server/backend/quiz/views.py

```
@@ -56,6 +56,14 @@ def post(self, request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    
+class UpdateDeleteQuizView(APIView):
+    permission_classes=[IsAuthenticated, IsAdminUser]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)